FF1Blazorizer/Pages/Randomize.razor

```diff
@@ -425,6 +425,15 @@
                         </tr>
                     </table>
                 </div>
+                <br />
+                <h4>Party Balance</h4>
+                <div class="col-md-12">
+                    <CheckBox UpdateToolTip="@UpdateToolTipID" Id="ChangeMaxMP" @bind-Checked="@Flags.ChangeMaxMP">Change Class Max MP</CheckBox>
```

Comment:
is there a reason this isnt tri-state?

its useful for the weekly challenge stuff so people dont know what to expect, or for people who just like torture like that